.github/settings.yml

```diff
@@ -0,0 +1,25 @@
+# The settings are inherited from common repository repo-settings (same file name and location)
+# You can append settings that are deeply merged with the inherited settings.
+#
+# When you have installed the GitHub App "repo-settings" in this repository,
+# any change of this settings.yml file is detected by the GitHub App and
+# the settings of this repository are updated immediately.
+#
+_extends: repo-settings:.github/toolkit-settings.yml
+
+# repo-specific settings
+#
+repository:
+  # See https://terraform-ibm-modules.github.io/documentation/#/implementation-guidelines?id=module-names-and-descriptions
+
+  # By changing this field, you rename the repository.
+
+  # Uncomment this name property and set the name to the current repo name.
+  name: "terraform-ibm-toolkit-object-storage"
+
+  # The description is displayed under the repository name on the
+  # organization page and in the 'About' section of the repository.
+
+  # Uncomment this description property
+  # and update the description to the current repo description.
+  description: "Terraform module to provision Object Storage in the IBM Cloud account"
```

Comment:
-> Provisions an IBM Cloud Object Storage instance

This aligns better with the conventions in term of description. Interested in @SirSpidey perspective as well.